core/src/main/java/org/mule/runtime/core/execution/ModuleFlowProcessingPhaseTemplate.java

```diff
@@ -39,6 +41,8 @@
    */
   Event routeEvent(Event event) throws MuleException;
 
+  Publisher<Event> dispatchEvent(Event event);
```

Comment:
should the introduction of this method call for deprecation of  Event routeEvent(Event) ?